src/main/kotlin/com/exactpro/th2/ws/client/Main.kt

```diff
@@ -133,7 +136,7 @@ fun run(
         handler,
         onMessage,
         onEvent
-    ).apply { registerResource("client", ::close) }
+    ).apply { registerResource("client", ::stop) }
 
     val listener = MessageListener<MessageGroupBatch> { _, message ->
```

Comment:
each conn in th2 is a common resource, and every new message to send should reestablish a connection to the remote system if it is needed.